### PR TITLE
G11 - KRAUT SPACE MAGIC

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## Changelog
:cl:
add: Armadyne has recently received stock of the G11.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/sec_haul/code/guns/bullets.dm [hunk 1]
```diff
@@ -295,3 +295,72 @@
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/projectiles.dmi'
 	icon_state = "smartgun_embed"
 	embedding = null
+
+/////////////4.73x33mm CASELESS
+//RUBBER | LETHAL | HP | IHDF
+///////////////////////////////
+/obj/item/ammo_casing/caseless/b473
+	name = "4.73x33mm bullet"
+	desc = "A 9mm bullet."
```
Comment (FlamingLily) [inline modular_skyrat/modules/sec_haul/code/guns/bullets.dm]:
```suggestion
	desc = "A 4.73x33mm bullet."
```

File: modular_skyrat/modules/sec_haul/code/guns/bullets.dm [hunk 2]
```diff
@@ -295,3 +295,72 @@
...
+	ricochet_incidence_leeway = 0
+	ricochet_chance = 130
+	ricochet_decay_damage = 0.8
+	shrapnel_type = null
+	sharpness = NONE
+	embedding = null
+
+/obj/item/ammo_casing/caseless/b473/ihdf
+	name = "4.73x33mm IHDF bullet"
+	desc = "A 9mm intelligent high-impact dispersal foam bullet casing."
```
Comment (FlamingLily) [inline modular_skyrat/modules/sec_haul/code/guns/bullets.dm]:
```suggestion
	desc = "A 4.73x33mm intelligent high-impact dispersal foam bullet casing."
```

File: modular_skyrat/modules/sec_haul/code/guns/guns.dm [hunk 3]
```diff
@@ -880,3 +880,54 @@
 	new /obj/item/ammo_box/magazine/smartgun(src)
 	new /obj/item/ammo_box/magazine/smartgun(src)
 	new /obj/item/ammo_box/magazine/smartgun(src)
+
+//////////////////////////////////////////////////////////
+/////////////////KRAUT SPACE MAGIC G11////////////////////
+//////////////////////////////////////////////////////////
+
+/obj/item/gun/ballistic/automatic/g11
+	name = "\improper Armadyne G11 K2"
```
Comment (Draganfrukts) [inline modular_skyrat/modules/sec_haul/code/guns/guns.dm:889]:
```suggestion
	name = "\improper Armadyne G11 K2 Mk. VI"
```
NO MOM IT'S NOT THE 1980 GUN IT'S A REVISED VERSION FROM 500 YEARS FORWARD!!!

**Review by Draganfrukts — CHANGES_REQUESTED** [review]
I accidentally started an entire review

File: modular_skyrat/modules/sec_haul/code/guns/guns.dm (hunk 3 above)
Comment (Gandalf2k15) [reply modular_skyrat/modules/sec_haul/code/guns/guns.dm:889]: Why don't we just pretend it's a new gun just designed by armafuck

**Pull request merged**